agent/runtime-attach/src/test/resources/applicationinsights.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
without this file, it will return Optional.empty. does the test still work without json config?

Reply:
The `RuntimeAttachTest` test will fail if we remove this file because _Application Insights_ needs a connection string provided in the json file. The connection string  may also be supplied with a system property or an environment variable, without a json file,  and the code execution path will then use the `Optional.empty()` line.